<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableEditDialog</class>
 <widget class="QDialog" name="VariableEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量增加变量</string>
  </property>
  <layout class="QVBoxLayout" name="VariableEditDialogLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabBase">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabBaseLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>批量增加变量</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>单位</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="editUnit">
            <property name="text">
             <string>mA</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editBatchNum">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>连续个数</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>数据类型</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>变量名</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>描述</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cboDataType">
            <item>
             <property name="text">
              <string>模拟量</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>数字量</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>寄存器间隔</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="editRegisterSpace">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="4">
           <widget class="QLineEdit" name="editDescription">
            <property name="text">
             <string>描述</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="editVariableName">
            <property name="text">
             <string>var</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDataAttribute">
      <attribute name="title">
       <string>数据属性</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabDataAttributeLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editMaxValue"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>最大值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>读写属性</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>初始值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cboAttribute">
            <item>
             <property name="text">
              <string>只读</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>只写</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>读写</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="editInitialValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>变比</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>最小值</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="editMinValue"/>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="editScaleValue">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIOConnect">
      <attribute name="title">
       <string>IO连接属性</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabIOConnectLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="chkIsIOConnect">
           <property name="text">
            <string>是否IO连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkIsSelfVariable">
           <property name="text">
            <string>是否内部变量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkGlobal">
           <property name="text">
            <string>全局属性</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>设备名</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cboDeviceName"/>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="cboTypeReadWrite">
            <item>
             <property name="text">
              <string>只读</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>只写</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>读写</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editRegisterAddress">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>读写类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>设备地址</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editDeviceAddress">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>最大值</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="editIOMaxValue"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>寄存器区</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cboRegisterSection"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>最小值</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="editIOMinValue"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>寄存器地址</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>初值</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="editIOInitValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>数据类型</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cboIODataType"/>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>变比</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="editIOScale">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_50">
            <property name="text">
             <string>偏移地址</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="editAddressOffset">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_51">
            <property name="text">
             <string>同帧地址</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="editAddressInFrame">
            <property name="text">
             <string>-1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
